src/lib/encoder.ts

```diff
@@ -424,7 +435,7 @@ export const valueContentEncodingMap = (valueContent: string) => {
 
           return padLeft(numberToHex(parsedValue), 64);
         },
-        decode: (value) => '' + hexToNumber(value),
+        decode: (value) => hexToNumber(value).toString(),
```

Comment:
Why converting it to a string? I think we should decode uintN data as number, not string.

Would suggest to remove this.

@Hugoo @b00ste

Reply:
It was converted to string before, instead of having `'' + hexToNumber(..)` i used the function `toString()`.

Reply:
I removed this behaviour in this PR afterwards.

So I guess we can already replace that here.

https://github.com/ERC725Alliance/erc725.js/pull/289

Reply:
Let's replace it in your next PR, because removing it breaks some tests